homeassistant/components/sensor/rmvtransport.py

```diff
@@ -77,22 +88,28 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
                 next_departure.get(CONF_TIME_OFFSET),
                 next_departure.get(CONF_MAX_JOURNEYS),
                 next_departure.get(CONF_NAME)))
-    add_entities(sensors, True)
+    async_add_entities(sensors, True)
 
 
 class RMVDepartureSensor(Entity):
     """Implementation of an RMV departure sensor."""
 
-    def __init__(self, station, destinations, directions,
+    def __init__(self, loop, session, station, destinations, directions,
                  lines, products, time_offset, max_journeys, name):
         """Initialize the sensor."""
+        self._loop = loop
+        self._session = session
         self._station = station
         self._name = name
         self._state = None
-        self.data = RMVDepartureData(station, destinations, directions, lines,
-                                     products, time_offset, max_journeys)
+        self.data = RMVDepartureData(loop, session, station, destinations,
+                                     directions, lines, products, time_offset,
+                                     max_journeys)
         self._icon = ICONS[None]
 
+        if self.data is None:
```

Comment:
How would this happen? Are there side effects, like I/O, when we instantiate the RMV data instance?

Reply:
No, this is just plain stupid.